packages/core/src/components/ErrorMessage/index.tsx

```diff
@@ -72,13 +67,29 @@ export default function ErrorMessage({
     >
       {message}
 
-      {id && (
+      {errorID && (
         <Spacing top={1}>
-          <MutedButton inverted onClick={createRedirectURL(id, url)}>
+          <MutedButton
+            inverted
+            openInNewWindow
+            href={error.error_url || Core.settings.errorURL.replace('{{id}}', errorID)}
+          >
             <T k="lunar.error.viewDetails" phrase="View error details" />
           </MutedButton>
         </Spacing>
       )}
+
+      {traceID && (
+        <Spacing top={1}>
+          <MutedButton
+            inverted
+            openInNewWindow
+            href={Core.settings.traceURL.replace('{{id}}', traceID)}
+          >
+            <T k="lunar.trace.viewDetails" phrase="View trace details" />
```

Comment:
Do I need to do anything with this new phrase?

Reply:
Yes you need to add it internally like our other apps.

Reply:
Done